backend/package.json

```diff
@@ -4,7 +4,19 @@
     "description": "The backend side for the Challedu project",
     "main": "index.js",
     "scripts": {
-        "start": "nodemon ./src/index.js"
+        "start": "nodemon ./src/index.js",
+        "precommit": "npx eslint --fix src/*.js"
```

Comment:
Since we have eslint installed as a dependency to the project, we can run it directly without npx. Lets also name this script 'lint' instead of precommit.